cmd/build-extensions-container.go

```diff
@@ -64,40 +64,30 @@ func buildExtensionContainer() error {
 	}
 	sha256sum := fmt.Sprintf("%x", hash.Sum(nil))
 
-	// Update the meta.json to include metadata for our OCI archive
-	metapath := filepath.Join(buildPath, "meta.json")
-	jsonFile, err := os.Open(metapath)
-	if err != nil {
-		fmt.Println(err)
-	}
-	defer jsonFile.Close()
-	jsonBytes, err := ioutil.ReadAll(jsonFile)
-	if err != nil {
-		return err
-	}
-	var cosaBuild cosa.Build
-	err = json.Unmarshal(jsonBytes, &cosaBuild)
-	if err != nil {
-		return err
-	}
-
 	cosaBuild.BuildArtifacts.ExtensionsContainer = &cosa.Artifact{
 		Path:            targetname,
 		Sha256:          sha256sum,
 		SizeInBytes:     float64(stat.Size()),
 		SkipCompression: true,
 	}
+	cosaBuild.MetaStamp = cosaBuild.MetaStamp + 1
 
 	newBytes, err := json.MarshalIndent(cosaBuild, "", "    ")
 	if err != nil {
 		return err
 	}
-
-	err = ioutil.WriteFile(metapath, newBytes, 0644)
+	extensions_container_meta_path := filepath.Join(buildPath, "meta.extensions-container.json")
+	err = ioutil.WriteFile(extensions_container_meta_path, newBytes, 0644)
 	if err != nil {
 		return err
 	}
-
-	fmt.Printf("Updated %s\n", metapath)
+	abs_new_json, err := filepath.Abs(extensions_container_meta_path)
+	if err != nil {
+		return err
+	}
+	cosaMeta := "cosa meta --build " + buildID + " --artifact extensions-container --artifact-json " + abs_new_json
```

Comment:
```suggestion
	cosaMeta := "cosa meta --build " + buildID + " --artifact-json " + abs_new_json
```

I think from reading the python code [here](https://github.com/coreos/coreos-assembler/blob/805ff1df68b8c3df2b4b91b763eb01bd129bebdb/src/cmd-meta#L98-L103) and [here](https://github.com/coreos/coreos-assembler/blob/805ff1df68b8c3df2b4b91b763eb01bd129bebdb/src/cosalib/meta.py#L141-L161) that passing `--artifact` here does nothing because we're not using `COSA_DELAYED_MERGE`. 

Can someone confirm my understanding is correct?

Reply:
I am testing it, to make sure it works.

Reply:
seems to work on the current commit.